p2p/net/connmgr/connmgr.go

```diff
@@ -173,7 +173,7 @@ func (cm *BasicConnMgr) memoryEmergency() {
 
 	// Trim connections without paying attention to the silence period.
 	for _, c := range cm.getConnsToCloseEmergency(target) {
-		log.Infow("low on memory. closing conn", "peer", c.RemotePeer())
+		log.Debugw("low on memory. closing conn", "peer", c.RemotePeer())
```

Comment:
This definitely shouldn't be hidden as debug. Running low on memory is a pretty serious condition.